.github/scripts/.helm-tests/default/result.yaml

```diff
@@ -7037,6 +7037,12 @@ spec:
     - jsonPath: .status.value
       name: Value
       type: string
+    - jsonPath: .spec.range.step
```

Comment:
it seems like the helm tests are failing because of the ordering of those fields - in the modified CRD, `step` and `aggregation` are added before ` value`, whereas here they are listed after `value` - if this is fixed, then all checks should pass